go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/vmware/terraform-provider-wavefront
 require (
 	github.com/WavefrontHQ/go-wavefront-management-api v1.14.0
 	github.com/google/go-cmp v0.5.2
-	github.com/hashicorp/terraform-plugin-sdk v1.11.0
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.4.3
```

Comment:
Interesting your PR title says v2.6.1 but this file says v2.4.3. Not sure how that happened.

Reply:
Done.

Reply:
Good catch. I upgraded to v2.6.1. The automated tools that I used must have been made when they were on v2.4.3